src/main/java/org/carlspring/strongbox/janusgraph/graph/gremlin/GremlinConfig.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.janusgraph.graph.gremlin;
+
+import java.io.InputStream;
+
+import org.apache.cassandra.service.CassandraDaemon;
+import org.apache.tinkerpop.gremlin.server.GremlinServer;
+import org.apache.tinkerpop.gremlin.server.Settings;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Configuration
+@ComponentScan
+public class GremlinConfig
```

Comment:
this configuration should be optional, like we currently have it in strongbox. So you can disable graph explorer and `GremlinServer` which provide remote protocol to get access to DB instance.